packages/openapi-typescript/examples/github-api.ts

```diff
@@ -73010,7 +73010,7 @@ export interface components {
             files?: string[];
             bin?: Record<string, never>;
             man?: Record<string, never>;
-            directories?: string | Record<string, never>;
+            directories?: string | Record<string, never> | null;
```

Comment:
This seems wrong, but is generated from:

```yaml
oneOf:
  - type: string
  - type: object
nullable: true
```

This is correct, at least as long as we generate `Record<string, never>` for empty objects (with no additionaalProperties)